jlm/hls/Makefile.sub

```diff
@@ -88,6 +88,7 @@ libhls_TESTS += \
 	tests/jlm/hls/backend/rvsdg2rhls/TestTheta \
 	tests/jlm/hls/backend/rvsdg2rhls/UnusedStateRemovalTests \
 	tests/jlm/hls/backend/rvsdg2rhls/test-loop-passthrough \
+	tests/jlm/hls/util/TestView \
```

Comment:
Please call the file `ViewTests`. If we prefix everything with `Test`, then alphabetical sorting is defeated.

Reply:
Renamed